sample.config.yaml

```
@@ -137,7 +137,11 @@ email:
   auth:
     username: janet
     password: foobar
-
+# Pindora smart lock API configuration
+pindora:
+  enabled: false
+  token: d217d336-550g-4dd3-b834-b7deb95b7607
+  id: 66c1b208-7b31-4747-a331-fc9753634104
```

I hope these are not real tokens. If they are, please reset them asap. It's better not to commit real looking tokens into example config files.

```suggestion
  token: secrettoken
  id: pindoraid
```